mava/logger.py

```diff
@@ -74,6 +75,8 @@ def log(
                 float(np.mean(episodes_length)),
                 t_env,
             )
+            logger.log_stat(prefix.lower() + "steps_per_second", steps_per_second, t_env)
+
             if trainer_metric:
```

Comment:
What do you think about adding a prefix to these info? I'm not sure if it would be beneficial.